ci/benchmark/build.sh

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
+# Copyright (c) 2018, NVIDIA CORPORATION.
+##########################################
+# cuGraph Benchmark test script for CI   #
+##########################################
+
+set -e
+set -o pipefail
+NUMARGS=$#
+ARGS=$*
+
+function logger {
+  echo -e "\n>>>> $@\n"
+}
+
+function hasArg {
+    (( ${NUMARGS} != 0 )) && (echo " ${ARGS} " | grep -q " $1 ")
+}
+
+function cleanup {
+  logger "Removing datasets and temp files..."
+  rm -rf $WORKSPACE/datasets/test
+  rm -rf $WORKSPACE/datasets/benchmark
+  rm -f testoutput.txt
+}
+
+# Set cleanup trap for Jenkins
+if [ ! -z "$JENKINS_HOME" ] ; then
+  logger "Jenkins environment detected, setting cleanup trap..."
+  trap cleanup EXIT
+fi
+
+# Set path, build parallel level, and CUDA version
+cd $WORKSPACE
+export PATH=/conda/bin:/usr/local/cuda/bin:$PATH
+export PARALLEL_LEVEL=4
+export CUDA_REL=${CUDA_VERSION%.*}
+export HOME=$WORKSPACE
+export GIT_DESCRIBE_TAG=`git describe --tags`
+export MINOR_VERSION=`echo $GIT_DESCRIBE_TAG | grep -o -E '([0-9]+\.[0-9]+)'`
+
+# Set Benchmark Vars
+export DATASETS_DIR=${WORKSPACE}/datasets
+export ASVRESULTS_DIR=${WORKSPACE}/ci/artifacts/asv/results
+export BENCHMARKS_DIR=${WORKSPACE}/benchmarks
+
+# Ensure ASV results directory exists
+
+mkdir -p ${ASVRESULTS_DIR}
+
+##########################################
+# Environment Setup                      #
+##########################################
+
+# TODO: Delete build section when artifacts are available
+
+logger "Check environment..."
+env
+
+logger "Check GPU usage..."
+nvidia-smi
+
+logger "Activate conda env..."
+source activate gdf
```

Comment:
the `gdf` environment has a symlink to keep it operational, but it might be better to use the new `rapids` conda environment.

src: https://github.com/rapidsai/gpuci-build-environment/blob/branch-0.15/rapidsai/base-runtime.Dockerfile#L59